docs/updating-nextflow.md

```diff
@@ -6,7 +6,9 @@ This page describes Nextflow release cadence, how to self-update Nextflow, and h
 
 ## Releases
 
-A stable version of Nextflow is released in the 4th and 10th month of each year. A edge version of Nextflow is released on a monthly basis. The edge version can be used to access the latest updates and experimental features. 
+A stable version of Nextflow is released in the 4th and 10th month of each year. An edge version of Nextflow is released on a monthly basis. The edge version can be used to access the latest updates and experimental features.
+
+Versions are numbered as `<year>.<month>.<patch>`. For example, `23.10.1` corresponds to the first patch of the October 2023 stable release.
 
```

Comment:
This scheme is named "Calendar versioning" https://calver.org/

Reply:
Thanks. I've linked this in the text. I think it's helpful to have that as an external reference.